Source/WebCore/html/HTMLImageElement.cpp

```diff
@@ -397,13 +397,6 @@ void HTMLImageElement::attributeChanged(const QualifiedName& name, const AtomStr
         if (isInTreeScope() && !m_parsedUsemap.isNull())
             treeScope().addImageElementByUsemap(m_parsedUsemap, *this);
         break;
-    case AttributeNames::compositeAttr: {
```

Comment:
You also need to remove this from `HTMLAttributeNames.in`.

Reply:
indeed! Thanks.

Reply:
removed.